test/checkpoint/test_checkpoint.c

```diff
@@ -237,6 +237,11 @@ main(int argc, char *argv[])
         return (EXIT_FAILURE);
     }
 
+    if (!g.use_timestamps && g.opts.disagg_storage) {
```

Comment:
while we are here can you add a check for named checkpoints (-c), column store (fix, col, mix), logged tables (-l), prepared transactions with a fix me (-p), predictable replay (-R) with a fix me?